Data_Clean_Up.ipynb

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
This shouldnt be a new jupyter notebook this should be a .py file it should be turned into a python module